server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/QGatesWs.java

```diff
@@ -79,8 +79,8 @@ static void addConditionParams(NewAction action) {
       .setPossibleValues(QualityGateConditionDto.ALL_OPERATORS);
 
     action.createParam(PARAM_PERIOD)
-      .setDescription("Condition period")
-      .setExampleValue("1");
+      .setDescription("Condition period. If not set, the absolute value is considered.")
```

Comment:
Another point : This parameter is now useless on differential metrics. It's not mandatory to deal it now, but you should keep it in mind for futur update of this WS.